README.md

```diff
@@ -1 +1,103 @@
-# CAPSTONE_2
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Horror Hour TV Shows
+
+This project contains codes that enables users get from and add data to the API. It shows information about movie shows. People can browse through, like and comment. The count of likes and count are registed.
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- VsCode
+- Webpack
+
+## Live Demo
+
+[Live Demo Link](https://ijeomaodinko.github.io/CAPSTONE_2/dist/)
+
+# Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+Open your console. Run the following 
+
+If you are using HTTPS
+
+    "git clone https://github.com/ijeomaodinko/CAPSTONE_2.git" 
+
+If you are using a SSH key.
+
+    "git clone git@github.com:ijeomaodinko/CAPSTONE_2.git" 
+
+
+Press ENTER to create your local clone. After creating it, you can open the page by opening with Visual Studio Code. 
+
+
+Typing    npm run start on the terminal and  open the dist folder and click on the index.html file.
+
+
+### Prerequisites
+- A modern web browser like Chrome, Safari, Firefox, etc.
+- JavaScript enabled.
+
+
+### Installation and Setup
+Install Node Modules ithrough the terminal 
+
+    npm install 
+
+
+### Build the Development Serve 
+
+    npm run build
+
+
+### Run Tests
+
+To run test on the test files used:
+
+    npm test 
+
+
+## Authors
+
+👤 **Ijeoma Odinko**
+
+- GitHub: [@githubhandle](https://github.com/ijeomaodinko)
+- Twitter: [@twitterhandle](https://twitter.com/iodinko)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ijeomaodinko)
+
+
+👤 **Vanessa Bonito**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL] This link is broken, So I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯